boost.sh

```
@@ -1,116 +1,49 @@
 package: boost
-version: "%(tag_basename)s"
-tag: "v1.83.0"
-source: https://github.com/alisw/boost.git
+version: "1.87.0"
+tag: "boost-1.87.0"
+source: https://github.com/boostorg/boost
 requires:
-  - GCC-Toolchain
-  - Python
+  - "GCC-Toolchain:(?!osx)"
```

Shellcheck (0.11.0)

[warning] This flag is used as a command name. Bad line break or missing [ .. ]?

(SC2215)